src/psyclone/psyir/transformations/loop_fuse_trans.py

```diff
@@ -66,19 +71,31 @@ def validate(self, node1, node2, options=None):
         :type node2: :py:class:`psyclone.psyir.nodes.Node`
         :param options: a dictionary with options for transformations.
         :type options: Optional[Dict[str, Any]]
+        :param bool options["force"]: whether to force fusion of the
+                                      target loop (i.e. ignore any dependence
+                                      analysis). This only skips a limited 
+                                      number of the checks, and does not
+                                      fully force merging.
 
-        :raises TransformationError: if one or both of the Nodes is/are not \
+        :raises TransformationError: if one or both of the Nodes is/are not
                                      a :py:class:`psyclone.psyir.nodes.Loop`.
         :raises TransformationError: if one or both Nodes are not fully-formed.
         :raises TransformationError: if the Nodes do not have the same parent.
-        :raises TransformationError: if the Nodes are not next to each \
+        :raises TransformationError: if the Nodes are not next to each
                                      other in the tree.
-        :raises TransformationError: if the two Loops do not have the same \
+        :raises TransformationError: if the two Loops do not have the same
                                      iteration space.
+        :raises TransformationError: if either Loop contains a reference to
+                                     the other Loop's variable and the Loops
+                                     don't share a loop variable.
```

Comment:
This isn't raised by this function I don't think, but instead by the static functions - should it still be in the docstring for this function?

Reply:
I like Transformation validate methods to document all possible fail conditions, since then is easy to read in the documentation what is supported and what not in a single place. So I would still add it here.

Also, arguably the two utility static methods should be private (underscore prefixed) and the Transformation user be unaware that they exist.

Reply:
Ok, I'll change accordingly.